tests/unit/core/single_event/test_likelihood.py

```
@@ -79,367 +110,99 @@ def test_initialization(self, detectors_and_waveform):
+        extra.set_psd(
+            PowerSpectrum.from_file(
+                str(FIXTURES_DIR / f"GW150914_psd_{extra.name}.npz")
+            )
         )
-        new_detector.set_psd(psd)
-        # Intentionally not setting data for this detector
-
-        ifos_mixed = ifos + [new_detector]
-
-        # Should raise ValueError mentioning the detector name
         with pytest.raises(ValueError, match="H1.*does not have initialized data"):
```

Ruff (0.15.13)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)